### PR TITLE
Padding footer

### PR DESCRIPTION
**Pull request by CyrilleKolle**
Link to ticket: https://fixa.itiden.se/project/1362/tickets/89
Was to update just contact footer but had to update for other pages since I passed margin as an initial props to page

### CODE REVIEW

File: components/Layout/Footer.tsx [hunk 1]
```diff
@@ -4,8 +4,14 @@ import tw from 'twin.macro';
 import { Logo } from '../Logo';
 import { SocialmediaGroupFooter } from '../SocialMediaIcons';
 
-const FooterContainer = styled.footer`
-  ${tw`w-full bottom-0 mb-16 mt-24 bg-gray-200 md:(bg-gray-900 mb-0 mt-32 border-solid border-0 border-t border-purple-700)`}
+interface IProps {
```
Comment (reekris) [inline components/Layout/Footer.tsx:7]: Can we call this `FooterProps` instead? Makes it clearer

File: components/Layout/Footer.tsx [hunk 2]
```diff
@@ -56,8 +62,8 @@ const Address = styled.div`
   ${tw`text-xs-footer text-gray-600 hover:text-purple-700 md:(text-gray-350 hover:text-teal-400)`};
 `;
 
-export const Footer = () => (
-  <FooterContainer>
+export const Footer = ({ margin = true }: IProps) => (
```
Comment (reekris) [inline components/Layout/Footer.tsx:65]: Can we rewrite this to `export const Footer: React.FC<FooterProps> = ({ margin = true }) => (`

File: components/Layout/Page.tsx [hunk 3]
```diff
@@ -7,6 +7,7 @@ import { Footer } from './Footer';
 
 interface PageProps {
   noLogoFnutt?: boolean;
+  margin?: boolean;
```
Comment (reekris) [inline components/Layout/Page.tsx:10]: Can we set the default value of margin to `true`? Then we don't have to add `margin={true}` to all elements

File: pages/kontakt.tsx [hunk 4]
```diff
@@ -70,11 +71,12 @@ const IndexPage: NextComponentType<{}, {}, IndexPageProps> = ({
   employees,
 }) => {
   return (
-    <Page>
+    <Page margin={false}>
       <IndexHeader title="/kontakt" />
       <TopContentWrapper>
         <QuarteredContent>
           <h1>Vi ser fram emot att höra från dig.</h1>
+          <ContactsGroup mail location phone />
```
Comment (reekris) [inline pages/kontakt.tsx:79]: Should this really be added? I thought we had removed this?

**Review by reekris — CHANGES_REQUESTED** [review]
Nice! Som changes needed

File: pages/kontakt.tsx [hunk 5]
```diff
@@ -38,10 +39,10 @@ const QuarteredContent = styled.div`
   }
 `;
 const GrayBanner = styled(Hero)`
-  ${tw`mt-16 py-10 bg-gray-600`}
+  ${tw`mt-16 py-10 bg-gray-700`}
 `;
 const GreenBanner = styled(Hero)`
```
Comment (reekris) [inline pages/kontakt.tsx:44]: Maybe change the name of this component since it is now grey?

**Review by ricllrstn — CHANGES_REQUESTED** [review]
Please also change color of the box and text closest to the footer.

**Review by reekris — APPROVED** [review]
Nice work!

**Review by ricllrstn — COMMENTED** [review]
Great work!

**Review by ricllrstn — APPROVED** [review]

**Pull request merged**